src/atc_api/main.go

```diff
@@ -32,9 +33,13 @@ func main() {
 		api.Log.Println("Connected to atcd socket on", args.ThriftAddr)
 	}
 
+	if args.IPv4 == "" || args.IPv6 == "" {
+		api.Log.Fatalln("You must provide both -4 and -6 arguments to run the API.")
```

Comment:
I think this is wrong. As much as we want to add v6 support, we want to support v4 or v6 only setups.

One could work around it by providing a dummy ipv6 ip, but then the api will still try to shape both ip flavours.... ending trying to connect to a bogus ip.